dev/src/view/TreeItemContext.ts

```diff
@@ -138,7 +138,7 @@ namespace TreeItemContext {
             contextValues.push(TreeItemContextValues.PROJ_RESTARTABLE);
         }
 
-        if (project.capabilities.metricsAvailable) {
+        if (project.capabilities.metricsAvailable || project.language === 'java') {
```

Comment:
fyi I've PR'd the Eclipse IDE version of this change at https://github.com/eclipse/codewind-eclipse/pull/493

Reply:
The change discussed on the call Tue 17 Dec 15:59 (GMT) is to add
```suggestion
        if (project.capabilities.metricsAvailable || ['java', 'nodejs'].includes(project.language) {
```